news/2023/2023-11-24-4dmwc-registrations-open.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: "4 Digit osu!mania World Cup 2024: Registrations Now Open!"
+date: 2023-11-24 15:00:00 +0000
+---
+
+The 5th edition of the 4 Digit osu!mania World Cup opens up. Because key-smashing tournaments shouldn't be exclusive to the upper echelon of players!
```

Comment:
```suggestion
The 5th edition of the 4 Digit osu!mania World Cup opens up, because key-smashing tournaments shouldn't be exclusive to the upper echelon of players!
```